examples/client/Locomotion/src/pages/ActiveRide/newMap.js

```diff
@@ -133,6 +139,15 @@ export default React.forwardRef(({
         });
       }
     }
+    if (currentBsPage === BS_PAGES.CONFIRM_FUTURE_RIDE) {
+      ref.current.fitToCoordinates(newFutureRide.stopPoints.map(sp => ({
```

Comment:
extract the fitToCoodinations to function and send the array, or create the array and call the function at the end of the main function